src/NuGet.Clients/NuGet.VisualStudio.OnlineEnvironment.Client/Properties/AssemblyInfo.cs

```diff
@@ -0,0 +1,32 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System.Reflection;
+using System.Runtime.InteropServices;
+using Microsoft.VisualStudio.Shell;
+
+// General Information about an assembly is controlled through the following 
+// set of attributes. Change these attribute values to modify the information
+// associated with an assembly.
+[assembly: AssemblyTitle("NuGet.VisualStudio.OnlineEnvironment.Client2")]
```

Comment:
Why Client2?

Reply:
Bug, I'll fix it.
I had to try 2 different ones :D